examples/wms/wms.md

```diff
@@ -0,0 +1,155 @@
+
+![Discovering WMS layers with QGIS](qgis-wms-layers.png)
+
+We can see that the OpenGeo demo WMS has a WMS layer named `ne:ne` with a basemap. Let's see how it looks:
+
+	var wmsLayer = L.tileLayer.wms('http://demo.opengeo.org/geoserver/ows?', {
+		layers: 'ne:ne'
+	}).addTo(map);
+
+<table><tr><td style='text-align: center'>
+<iframe src='wms-example1.html' width='616' height='416'></iframe>
+</td></tr><caption align='bottom'><small><a href='wms-example1.html'>See this example stand-alone.</a></small></caption></table>
+
+
+Or we can try the `nasa:bluemarble` WMS layer:
+
+	var wmsLayer = L.tileLayer.wms('http://demo.opengeo.org/geoserver/ows?', {
+		layers: 'nasa:bluemarble'
+	}).addTo(map);
+
+<table><tr><td style='text-align: center'>
+<iframe src='wms-example2.html' width='616' height='416'></iframe>
+</td></tr><caption align='bottom'><small><a href='wms-example2.html'>See this example stand-alone.</a></small></caption></table>
+
+
+The `layers` option is a comma-separated list of layers. If a WMS service has defined several layers, then a request for a map image can refer to more than one layer.
+
+For the example WMS server we're using, there is a `ne:ne_10m_admin_0_countries` WMS layer showing country landmasses and country names, and a `ne:ne_10m_admin_0_boundary_lines_land` WMS layer showing country boundaries. The WMS server will compose both layers in one image if we request both, separated with a comma:
+
+	var countriesAndBoundaries = L.tileLayer.wms('http://demo.opengeo.org/geoserver/ows?', {
+		layers: 'ne:ne_10m_admin_0_countries,ne:ne_10m_admin_0_boundary_lines_land'
+	}).addTo(map);
+
+Note this will request *one* image to the WMS server. This is different than creating a `L.TileLayer.WMS` for the countries, another one for the boundaries, and adding them both to the map. In the first case, there is one image request and it's the WMS server who decides how to compose (put on top of each other) the image. In the second case, there would be two image requests and it's the Leaflet webpage who decides how to compose them.
+
+If we combine this with the [layers control](/examples/layers-control.html), then we can build a simple map to see the difference:
+
+	var basemaps = {
+		Countries: L.tileLayer.wms('http://demo.opengeo.org/geoserver/ows?', {
+			layers: 'ne:ne_10m_admin_0_countries'
+		}),
+
+		Boundaries: L.tileLayer.wms('http://demo.opengeo.org/geoserver/ows?', {
+			layers: 'ne:ne_10m_admin_0_boundary_lines_land'
+		}),
+
+		'Countries, then boundaries': L.tileLayer.wms('http://demo.opengeo.org/geoserver/ows?', {
+			layers: 'ne:ne_10m_admin_0_countries,ne:ne_10m_admin_0_boundary_lines_land'
+		}),
+
+		'Boundaries, then countries': L.tileLayer.wms('http://demo.opengeo.org/geoserver/ows?', {
+			layers: 'ne:ne_10m_admin_0_boundary_lines_land,ne:ne_10m_admin_0_countries'
+		})
+	};
+
+	L.control.layers(basemaps).addTo(map);
```

Comment:
Yeah, I'll leave it like that, to make the code in the tutorial simpler. How the layers control works is detailed in its own tutorial.